charts/nodejs/templates/_helpers.tpl

```diff
@@ -18,7 +22,7 @@ Expand the name of the chart.
 {{- if (eq .Values.kubernetes.env "production") -}}
 {{- printf "%s.scp.biomage.net" (split "/" .Values.biomageCi.repo)._1 -}}
 {{- else -}}
-{{- printf "%s.scp-%s.biomage.net" (split "/" .Values.biomageCi.repo)._1 .Values.kubernetes.env -}}
+{{- printf "%s-%s-%s.scp.biomage.net" (split "/" .Values.biomageCi.repo)._1 .Values.kubernetes.env .Values.biomageCi.sandboxId -}}
```

Comment:
This was a bug. Instead of the URL being like `ui.scp-staging.bimage.net`, we will ahve it as `ui-staging-asd.scp.biomage.net` if we are in a non-prod deployment.